### DISCUSSION TITLE
Uno for Figma Preview not always correct

### DISCUSSION THREAD

**Discussion by swoestenburg** [unoplatform/uno#8817] [Q&A] [answered]
Hey!

I'm not sure if this is the right place, but I started using Uno for Figma and I noticed the preview isn't always right. Colors are off, elements are missing. I'm only using the asset components from the Toolkit as recommended. What could be wrong?

![Capture](https://user-images.githubusercontent.com/97455349/169108212-4d0ed8f0-1153-472c-9a2e-756c47bb7e60.PNG)

**Comment by NVLudwig** [comment, marked as answer]
Hello @swoestenburg,
Thank you for checking out the preview of the Uno Figma Toolkit.

Sorry to see things aren't generating as expected. We are working hard on an updated version of the UFT but I would be happy to take some time to have a look and troubleshoot this file for you. Could you share a copy of your Figma file with me so I can have a look at what is going on here?

Ludwig

**Reply by swoestenburg** [reply]
Hey @NVLudwig!

I've sent you an invitation to edit the project on the email address in your profile.

**Reply by NVLudwig** [reply]
Hello @swoestenburg,
Thanks for sharing your document with us. First let me say I'm impressed you managed to get everything setup properly without any help or documentation on our part. As I mentioned we are currently working on an update to this Toolkit preview and feedback like this from early adopters like yourself helps us a lot. Feel free to post more here as you encounter issues with the preview material.

Looking at the screenshot above I notice four specific issues I can address:

> 1. The font color not showing up properly in the plugin preview: it's du to the OnPrimary color being improperly named as a "Brush" when it is not in fact a brush, I corrected this in your Figma document it's now named OnPrimaryColor and works fine;
> 
> 2. The photo previews are not the images in Figma: that's currently expected behaviour in the plugin, we show mock data for images at the moment;
> 
> 3. Progress ring not rendering: I believe it's a bug in the plugin and am going to log it as such so we can correct it shortly;
> 
> 4. FAB not rendering: I believe it's also a plugin bug and am going to log it as such so we can correct this shortly;

Once the fix is applied the plugin should self update on re-launch. I will notify you when the new version is published (very shortly).

I hope this helps.

Ludwig

**Reply by swoestenburg** [reply]
Hey Ludwig,

Thanks for your efforts! I will close this Q&A for now, as I feel really helped by your answers. Looking forward to the new version, as I intend to continue using Uno.

**Reply by NVLudwig** [reply]
Hello again @swoestenburg,
Just letting you know we have a new version out of both the Figma file and the Plugin.
https://www.figma.com/community/file/1110792522046146058

I hope this resolves some of the issues you were having with the previous version.

Enjoy!
